maven/src/it/redeploy-release-artifact-allowed/verify.groovy

```diff
@@ -1,6 +1,6 @@
 import org.carlspring.strongbox.client.ArtifactClient
 
-def client = ArtifactClient.testInstance;
+def client = ArtifactClient.testInstanceLoggedInAsAdmin;
```

Comment:
I seem to have missed you dropping the method from the `strongbox-rest-client`. Could you please restore it and update the PR? Thanks. I'll close my PR.